__tests__/api.test.js

```diff
@@ -59,4 +59,45 @@ describe("Returns with the correct JSON endpoint", () => {
         expect(input).toMatchObject(expectedOutput);
       });
   });
+  test("/api/reviews/:review_id", () => {
+    const review_id = 3;
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
+      .expect(200)
+      .then((response) => {
+        response.body.review.forEach((review) => {
+          expect(typeof review.title).toBe("string");
+          expect(typeof review.created_at).toBe("string");
+          expect(typeof review.votes).toBe("number");
+          const input = Object.keys(
+            endpoint["GET /api/reviews/:review_id"].exampleResponse
+          );
+          const expectedOutput = Object.keys(response.body);
+          expect(input).toMatchObject(expectedOutput);
+        });
+      });
+  });
+  test("/api/reviews", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((response) => {
+        response.body.reviews.forEach((review) => {
```

Comment:
I would recommend an assertion above the .foreach to check the length of the array, to check /api/reviews gives you the exact number of reviews you are expecting to see